gum/arch-x86/gumx86writer.c

```diff
@@ -2373,10 +2373,8 @@ gum_get_jcc_opcode (x86_insn instruction_id)
     case X86_INS_JG:
       return 0x7F;
     case X86_INS_JCXZ:
-      return 0xE3;
-    case X86_INS_JECXZ:
-      return 0xE3;
     case X86_INS_JRCXZ:
+    case X86_INS_JECXZ:
```

Comment:
- Would be good to preserve the order like it was.
- Hexdecimal constants should be lower-case for consistency.